docs/installation_configuration_guide/DeploymentManagerConfiguration.md

```diff
@@ -9,20 +9,21 @@ Type of Deployment Manager is defined with `type` parameter, e.g. for running sc
 deploymentConfig {     
   type: "flinkStreaming"
   restUrl: "http://localhost:8081"
+  # additional configuration goes here
 }
 ```
 
 Look at [configuration areas](./#configuration-areas) to understand where Deployment Manager configuration should be placed in Nussknacker configuration.
 
-## Lite engine based on Kubernetes
+## Kubernetes native Lite engine configuration
                                                                                 
 Please remember, that K8s Deployment Manager has to be run with properly configured K8s access. If you install the Designer
 in K8s cluster (e.g. via Helm chart) this comes out of the box. If you want to run the Designer outside the cluster, you 
 have to configure `.kube/config` properly.
 
-Both processing modes: `streaming` and `request-response` share the majority of configuration.
+With the exception of the `servicePort` configuration option, all remaining configuration options apply to both `streaming` and `request-response` processing modes.
 
-`lite-k8s` Deployment Manager has the following configuration options:                 
+The table below contains configuration options for the Lite engine. If you install Designer with Helm, you can customize the Helm chart to use different values for those options. If you install Designer outside of the K8s cluster then the required changes should be applied under the `deploymentConfig` key as any other Nussknacker non K8s configuration. 
```

Comment:
maybe link to https://artifacthub.io/packages/helm/touk/nussknacker#configuration-in-values-yaml?

Reply:
Added the link